### PR TITLE
docs: collecting additional logs from the node using OTC

### PR DESCRIPTION
**Pull request by dmolenda-sumo**

### CODE REVIEW

File: docs/best-practices.md [hunk 1]
```diff
@@ -290,7 +290,59 @@ sumologic:
...
+        receivers:
+          otlp/extrafiles:
+            protocols:
+              http:
+                endpoint: 0.0.0.0:4319
+        service:
+          pipelines:
+            logs/extrafiles:
+              receivers: [otlp/extrafiles]
+              exporters: [sumologic/containers]
```
Comment (rnishtala-sumo) [inline docs/best-practices.md:340]: should this be `sumologic/node` or `sumologic/systemd` instead?

Also should we be recommending using a few processors like these?
```
Batch Processor
Memory Limit Processor
Attributes Processor
Resource Processor
```

File: docs/best-practices.md [hunk 2]
```diff
@@ -290,7 +290,59 @@ sumologic:
...
+
+The following configuration can be used for these purposes:
+
+```yaml
+otellogs:
+  config:
+    merge:
+      receivers:
+        filelog/extrafiles:
+          include: [/var/log/extrafiles/extrafile.log]
```
Comment (ballensans) [inline docs/best-practices.md:307]: Can you also provide an example for receiving data from the SystemD Journal?  In particular, this approach looks very close to supporting collection of IPTables "LOG" directives.

**Review by astencel-sumo — APPROVED** [review]
This is great :+1:

**Review by rnishtala-sumo — APPROVED** [review]

File: docs/best-practices.md (hunk 2 above)
Comment (swiatekm-sumo) [reply docs/best-practices.md:307]: systemd logs are already collected by default, you don't need any additional configuration for those.

File: docs/best-practices.md (hunk 1 above)
Comment (swiatekm-sumo) [reply docs/best-practices.md:340]: I think we should have the batch processor and memory limiter here, but no need to do anything else by default.

**Pull request merged**